docs/monitors/df.md

```diff
@@ -0,0 +1,79 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# df
+
+
+This monitor reports metrics about free disk space.
+
+```yaml
+monitors:
+ - type: df
+```
+
+
+Monitor Type: `df`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/df)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: **No**
```

Comment:
This should be "Yes" since it is possible that people would want to configure this multiple times with mutually exclusive sets of filesystems.